content/blog/2020-05-04-dvc-3-years-and-1-0-release.md

```diff
@@ -0,0 +1,238 @@
+
+The data science and ML space is fast-paced and vibrant, and we're proud that
+DVC is making an impact on discussions about best practices for healthy,
+sustainable ML. Every week, we chat with companies and research groups using DVC
+to make their teams more productive. We're proud to be part of the growing MLOps
+movement: so far, a majority of CI/CD for ML projects are implemented with DVC
+under the hood.
+
+I can confidently say that DVC wouldn't have been possible without a lot of help
+from our community. Thank you to everyone who has supported us:
+
+**DVC core team.** The DVC team has been the force driving our project's
+evolution - we've grown from 2 to 12 full-time engineers, developers, and data
+scientists. Half of the team is purely focus on DVC while the other half on
+related to DVC new projects. We often get feedback about how fast our team
+answers user questions - we've been told our user support is one of DVC's
+"killer features". It's all thanks to this amazing team.
+
+**DVC contributors.** As of today, the DVC code base has
+[126 individual contributors](https://github.com/iterative/dvc/graphs/contributors).
+Many of these folks put hours into their code contributios. We're grateful for
+their tenacity and generosity.
+
+![](/static/uploads/images/2020-05-04/vera-sativa.png)_Vera - 100th DVC
+contibutor [on GitHub](https://github.com/verasativa/)._
+
+**Documentation contributors.** Another
+[124 people contributed](https://github.com/iterative/dvc.org/graphs/contributors)
+to the [DVC documentation](https://dvc.org/doc) and
+[the website](https://dvc.org/). Every time a new person tries out DVC, they
+benefit from the hard work that's gone into our docs.
+
+**Active community members.** Active DVC users help our team understand and
+better anticipate their needs and identify priorities for development. They
+share bright ideas for new features, locate and investigate bugs in code, and
+welcome and support new users.
+
+**People who give DVC a shot.** Today, there are thousands of data scientists,
+ML engineers, and developers using DVC on a regular basis. The number of users
+is growing every week. Our [Discord channel](http://dvc.org/chat) has almost two
+thousand users. Hundreds more connect with us through email and Twitter. To
+everyone willing to try out DVC, thank you for the opportunity.
+
+## DVC 1.0. is the result of 3 years of learning
+
+All these contributions, big and small, have a collective impact on DVC's
+development. I'm happy (and a bit nervous) to announce that a pre-release of a
+brand new DVC 1.0 is ready for public beta testing.
+
+The new DVC is inspired by discussions and contributions from our community -
+both fresh ideas and bug reports 😅.
+
+Here are the most significant features we’re excited to be rolling out soon:
+
+### [Run cache](https://github.com/iterative/dvc/issues/1234)
+
+_Learnings:_ Forcing users to make Git commits for each ML experiment creates
+too much overhead.
```

Comment:
Also: Should this one be listed after multi-stage pipelines file? Maybe even after `dvc plot`.

Reply:
@jorgeorpinel this is the strongest one, I think

Reply:
@jorgeorpinel, try it out and you'll know. This will make you fall in love with dvc all over again. :smile:

Reply:
Will do!